autoextract_spiders/spiders/crawler_spider.py

```diff
@@ -223,11 +223,17 @@ def _schedule_seed_urls(self, seed_urls):
                 continue
             # Initial request to the seed URL
             self.crawler.stats.inc_value('x_request/seeds')
-            yield Request(url,
+            request = Request(url,
                           meta={'source_url': url},
                           callback=self.main_callback,
                           errback=self.main_errback,
                           dont_filter=True)
+            # Trick required to avoid some seeds to be never processed or too late.
+            try:
+                self.crawler.engine.crawl(request, self)
+            except AssertionError:
```

Comment:
Out of curiosity, what are the conditions to trigger the assertion error?

Reply:
To be honest, no idea. I copied this fix to get all initial URLs fetched form other projects. @kmike  do you know which conditions raise this assertion error?

Reply:
I recall it fails always if you remove this condition; something about engine not started, but I'm not sure.

Reply:
Okay, that makes some sense, thanks guys